Framework/Indexing/src/Group.cpp

```diff
@@ -26,9 +26,9 @@ IndexInfo group(const IndexInfo &source, std::vector<SpectrumNumber> &&specNums,
                              "number and grouping vectors");
   std::vector<SpectrumDefinition> specDefs;
   const auto &sourceDefs = source.spectrumDefinitions();
-  for (const auto &group : grouping) {
+  for (const auto &groupIndices : grouping) {
     specDefs.emplace_back(SpectrumDefinition{});
-    for (const auto &i : group) {
+    for (const auto &i : groupIndices) {
```

Comment:
If we're changing variable names for clarity, could I suggest changing `i` to `groupIndex` to be slightly clearer about its use

Reply:
Good idea.